conf/messages

```diff
@@ -1404,12 +1406,16 @@ businessdetails.lettersaddress.title = Is this the right address to send you let
 tradingpremises.whatyouneed.paragraph1.text = We need to know where you trade from, and where your agents trade from. An agent is an individual or business that works on behalf of your business, to offer your services.
 tradingpremises.whatyouneed.requiredinfo.heading = For each of your trading premises you’ll be asked things like:
 tradingpremises.whatyouneed.requiredinfo.text.1 = trading name and address
-tradingpremises.whatyouneed.requiredinfo.text.2 = what the business does from these premises
-tradingpremises.whatyouneed.requiredinfo.text.3 = date you’ll start trading from these premises
-tradingpremises.whatyouneed.requiredinfo.text.4 = if it is also a residential address
-tradingpremises.whatyouneed.agents.sub.heading = Agents
-tradingpremises.whatyouneed.agents.desc.1 = If you use agents, we also need to know where your agents trade from.
-tradingpremises.whatyouneed.agents.desc.2 = An agent is an individual or business that works on behalf of your business to provide your services. An employee is not an agent.
+tradingpremises.whatyouneed.requiredinfo.text.2 = the date you started or will start trading from these premises
+tradingpremises.whatyouneed.requiredinfo.text.3 = if it is a residential address
+tradingpremises.whatyouneed.requiredinfo.text.4 = the services your business provides at these premises
+tradingpremises.whatyouneed.requiredinfo.text.5 = which money service business activities your business does at these premises
+tradingpremises.whatyouneed.requiredinfo.text.6 = company name and registration number
+tradingpremises.whatyouneed.requiredinfo.text.7 = name and date of birth
+tradingpremises.whatyouneed.requiredinfo.text.8 = business partners
+tradingpremises.whatyouneed.agents.sub.heading = Agents’ trading premises
```

Comment:
Is that punctuation mark in the right place?

Reply:
yes

Reply:
Should it not be **Agent’s** ?

Reply:
If this is me not being grammatically correct I am sorry. I will check with Lisa. :-)